package.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "gondolin-frontend",
+  "version": "1.0.0",
+  "description": "The gondolin frontend.",
+  "license": "BSD-2-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/terrestris/gondolin-client"
+  },
+  "keywords": [
+    "gondolin"
+  ],
+  "private": true,
+  "author": "terrestris GmbH & Co. KG",
+  "contributors": [
+    {
+      "name": "Kai Volland",
+      "email": "volland@terrestris.de",
+      "url": "https://github.com/kaivolland"
+    },
+    {
+      "name": "Lukas Lohoff",
+      "email": "lohoff@terrestris.de",
+      "url": "https://github.com/lukaslohoff"
+    }
+  ],
+  "main": "index.js",
+  "scripts": {
+    "pretest": "npm run lint",
+    "test": "mocha --compilers js:babel-register --recursive 'app/spec/**/*.test.js*'",
+    "coverage": "istanbul cover _mocha -- --compilers js:babel-register --colors --reporter dot --recursive 'app/spec/**/*.test.js*'",
+    "lint": "eslint --ext .jsx,.js app/**",
+    "lint-fix": "eslint --fix --ext .jsx,.js app/**",
+    "start:dev": "webpack-dev-server --content-base app/ --hot --progress",
+    "prebuild": "npm run test && rimraf ./build/* && npm run build-ol",
+    "build:production": "NODE_ENV=production webpack --optimize-minimize --progress --config webpack.config.js --mode production",
+    "build:test": "webpack --optimize-minimize --progress --config webpack.config.js"
+  },
+  "dependencies": {
```

Comment:
Please check if you can remove some dependencies as they are not used anymore... should be a lot.